### PR TITLE
Fixed broken link in git-workflow.md

### PR DESCRIPTION
**Pull request by alanna-blinn**
Updated broken URL to Getting Started with GitHub Desktop

### CODE REVIEW

File: docs/04-how-we-work/tools/google-hangouts.md [hunk 1]
```diff
@@ -7,22 +7,22 @@ Google Hangouts are video chats in the browser. Here are some tips for using Han
...
 *   Be on time - please! Every 10 seconds actually matters
 *   Default to video on for all video meetings
 *   Add a photo to your account so it shows up in Hangout if your camera is off
-*   We use the chat and screensharing features often
+*   We use the chat and screen sharing features often
 *   Turn off your camera or reduce bandwidth if connection is poor
 *   Use some kind of headset or headphones instead of relying solely on computer mic (computer mics can sometimes create echoes)
 *   Use proper etiquette for Hangout professionalism (consider the appropriateness of background visuals/noise, attire/dress, use of effects, eating, muting, etc.)
-*   It's okay to mute other each other
+*   It's okay to mute yourself
```
Comment (openprivacy) [inline docs/04-how-we-work/tools/google-hangouts.md]: I believe this was meant to be "It's okay to mute others"

**Review by openprivacy — COMMENTED** [review]
Looks great! @alanna-casey will make one update to manage https://github.com/CivicActions/handbook/pull/533/files#r377175996 and then this is ready to merge.

**Review by openprivacy — APPROVED** [review]
All looks good - thanks, Alanna!

File: docs/01-welcome-to-civicactions/training/git-ssh.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ Host *.civicactions.net
 Port 940
 ```
 
-## Gitlab and SSH
+## GitLaband SSH
```
Comment (grugnog) [inline docs/01-welcome-to-civicactions/training/git-ssh.md]: I think we need the space in here :)

**Review by grugnog — CHANGES_REQUESTED** [review]
Looks great - thanks for working on this!
Found one minor typo (noted in review).

File: docs/01-welcome-to-civicactions/training/git-ssh.md (hunk 2 above)
Comment (openprivacy) [reply docs/01-welcome-to-civicactions/training/git-ssh.md]: Heh - I think I read that upside down in my review. @alanna-casey please update.

**Review by grugnog — APPROVED** [review]

**Pull request merged**